backend/charts/api/templates/api-env-vars.yaml

```diff
@@ -26,7 +26,7 @@ stringData:
     NUCLEUS_ENV: {{ .Values.nucleusEnv }}
     {{- end }}
 
-    SHUTDOWN_TIMEOUT_SECONDS: {{ .Values.shutdownTimeoutSeconds | quote }}
+    SHUTDOWN_TIMEOUT_SECONDS: {{ .Values.shutdownTimeoutSeconds | default "300" | quote }}
```

Comment:
One other way we could go with this is to only add the shutdown timeout to the deployment if the value is present. That way the deployment will just defer to the Kubernetes default instead of our hardcoded one.
wdyt?

Reply:
I think you're right, that's a better solution! I'll make the change shortly. Thanks!